database/factories/OnlineMailableFactory.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Database\Factories;
+
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Sammyjo20\Wagonwheel\Models\OnlineMailable;
+
+
+class OnlineMailableFactory extends Factory
+{
+    protected $model = OnlineMailable::class;
+
+    public function definition()
+    {
+        $expiration = config('wagonwheel.message_expires_in_days', 30);
+
+        return [
+            'uuid' => $this->faker->uuid,
+            'expires_at' => Carbon::now()->addDays($expiration),
+            'content' => $this->faker->sentence(6),
+        ];
+    }
+
+    public function expired()
+    {
+        return $this->state([
+            'expires_at' => Carbon::parse('1 year ago'),
+        ]);
+    }
+
+    public function expiresIn(string $expirePeriod)
```

Comment:
Could this accept a `DateTimeInterface` instead of a `string` so that we can pass through instances of `Carbon`?  That way,  the `expired()` state method could then do:

```php
return $this->expiresIn(
    now()->subYear()
);
```

Reply:
Sure thing! Actually, I first implemented it that way, but then thought it would be cool to use a string in your tests. 

However, I think you're right to use the other approach 👍

Reply:
@ryangjchandler addressed in commit ab44d73